code/modules/power/antimatter/shielding.dm

```diff
@@ -29,12 +29,17 @@ proc/cardinalrange(var/center)
 	var/efficiency = 1//How many cores this core counts for when doing power processing, plasma in the air and stability could affect this
 	var/coredirs = 0
 	var/dirs=0
+	var/mapped=0 //Set to 1 to ignore usual suicide if it doesn't immediately find a control_unit
```

Comment:
Silly nitpick, but add a space in between the =s here if you get the chance, to var/mapped = 0